scripts/proto-tools-installer.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+set -ue
```

Comment:
wow! very nice bash code here! 
kind of a nit, but could include a check to see if protoc is already installed, but it wouldn't matter at all if we didnt

Reply:
maybe something like:
```
which protoc > /dev/null
if [ $? == 1  ]; then
  return 1
fi
  
```